delta-app/src/main/java/io/cdap/delta/app/service/common/AbstractAssessorHandler.java

```diff
@@ -64,6 +64,7 @@ protected void respond(String namespaceName, HttpServiceResponder responder, Nam
     try {
       NamespaceSummary namespaceSummary = context.getAdmin().getNamespaceSummary(namespaceName);
       if (namespaceSummary == null) {
+        LOG.error(String.format("Namespace '%s' not found", namespaceName));
```

Comment:
We should use the logger's build in parameter substitution
```suggestion
        LOG.error("Namespace {} not found", namespaceNam);
```